product-development/README.md

```diff
@@ -111,41 +113,38 @@ When a section is done, replace :white_small_square: with :white_check_mark:
-affirmatively articulate what epics in the backlog they want to work on and why.
+The kickoff meeting involves the following process:
+- The product manager explains the business need
+- Contributors outline their thoughts in the following structure:
+	- "What do you think the team should be working on?"
+	- "What do you think you should be working on?"
+	- "What do you think could go wrong?"
+	- "What is this discussion missing?"
+- Once these are outlined they are first reviewed to triage things that might be out of scope of the business need, or concerns that might be off topic
+- Then every contributor goes over their points, and these are sorted into work units or concerns as appropriate
+- As part of the discussion, taking into account available bandwidth and what a minimum viable deliverable should be, the team should define:
+	- How do we know we're done with this effort? (What's in and out of scope?)
+	- What does a win look like? (How are we measuring success? How can we celebrate it? What does a failure look like?)
 
-Every two weeks, Nick will talk to you your impressions of the backlog, and try
-to get a sense of what you think is important and want to work on.
+After the kickoff meeting the project manager updates the 'workstream status' document to include the decisions made in the kickoff meeting, including any of: updated business goal, scope, out-of-scope, definition of done, metrics, concerns, and so on. The project manager then organizes the work units and assigns contributors to them, and work begins.
 
-Nick will assign you to a team working on an ongoing epic, or a new epic near
-the top of the backlog, trying to balance preferences and current priority order.
+Regularly, at a frequency defined by the group in question, 'retro-planning' meetings take place. During these, the work that has already taken place is reviewed in terms of happy, mixed, and sad feelings, any tensions are discussed, and any new ideas stemming from these are then integrated into the workstream status document.
 
-If there's an epic that's high in priority order but no one wants to work on,
-then Nick should dig more into why. Do we need to do more research upfront?  Is
-the epic defined incorrectly? Is it misaligned with what we're currently
-focusing on?
+Once the workstream reaches its conclusion a wrap up meeting takes place. It contains the retro element from the retro-planning meeting, plus a review of goals achieved, metrics, how this should inform future work, and so on.
 
-There may be rare cases where someone gets assigned to an epic that they
-don't want to work on, but in those cases we should explicitly call out
-that this is happening and why.
+Misc:
+- As a rule of thumb, a contributor should work on only -one- workstream at a time.
+- We are deliberately not concerned with deadlines, but with addressing business concerns. (In other words: we work on something until we're happy with it, and dates do not matter.)
```

Comment:
one nit: we also should mark something done when we've decided it's not solving the business concern, and have decided to table it!

Reply:
ok fixed